docs/concepts/storage/volumes.md

```diff
@@ -696,6 +697,15 @@ spec:
 
 For further detail, plese the see the [ScaleIO examples](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/volumes/scaleio).
 
+### local
+This volume type is alpha in 1.7.
+
+A `local` volume represents a mount point or directory that has node affinity.
```

Comment:
I am not sure that users would understand what node affinity means. I would appreciate more "user-friendly" description, like "local volume represents a mounted local storage device such as a disk, a partition or even a local directory". And that scheduler makes sure that pods that use a local volume are scheduled to the right nodes.

In addition, some comparison with HostPath volumes would be nice. When should the user use HostPath and when a local volume?